VTExts.md

```diff
@@ -0,0 +1,386 @@
+
+---
+
+### `FARTTY_INTERACT_CLIPBOARD` ('c')
+
+Namespace for all clipboard-related operations.
+
+-   **In Stack:**
+    | Argument     | Type   | Description                                         |
+    | ------------ | ------ | --------------------------------------------------- |
+    | Sub-command  | `char` | A character identifying the specific clipboard action. |
+    | ...          | ...    | Arguments specific to the sub-command.              |
+-   **Out Stack:** Varies by sub-command.
+
+#### Clipboard Operations
+
+##### `FARTTY_INTERACT_CLIP_OPEN` ('o')
+Authorizes and opens the clipboard for subsequent operations.
+
+-   **In Stack:**
+    | Argument     | Type     | Description                                         |
+    | ------------ | -------- | --------------------------------------------------- |
+    | Passcode     | `string` | A unique client identifier (32-256 chars).         |
+-   **Out Stack:**
+    | Argument          | Type       | Description                                                                      |
+    | ----------------- | ---------- | -------------------------------------------------------------------------------- |
+    | Status            | `int8_t`   | `1` for success, `0` for failure, `-1` for access denied.                          |
+    | Server Features   | `uint64_t` | (Optional) A bitmask of `FARTTY_FEATCLIP_*` flags supported by the server.         |
+
+##### `FARTTY_INTERACT_CLIP_CLOSE` ('c')
+Closes the clipboard, finalizing the transaction.
+
+-   **In Stack:** None.
+-   **Out Stack:**
+    | Argument   | Type     | Description                                         |
+    | ---------- | -------- | --------------------------------------------------- |
+    | Status     | `int8_t` | `1` for success, `0` for failure, `-1` if not open.   |
+
+##### `FARTTY_INTERACT_CLIP_EMPTY` ('e')
+Clears the clipboard content.
+
+-   **In Stack:** None.
+-   **Out Stack:**
+    | Argument   | Type     | Description                                         |
+    | ---------- | -------- | --------------------------------------------------- |
+    | Status     | `int8_t` | `1` for success, `0` for failure, `-1` if not open.   |
+
+##### `FARTTY_INTERACT_CLIP_ISAVAIL` ('a')
+Checks if a specific data format is available on the clipboard.
+
+-   **In Stack:**
+    | Argument   | Type       | Description                                         |
+    | ---------- | ---------- | --------------------------------------------------- |
+    | Format ID  | `uint32_t` | The format to check (e.g., `1` for CF_TEXT).        |
+-   **Out Stack:**
+    | Argument   | Type     | Description                                         |
+    | ---------- | -------- | --------------------------------------------------- |
+    | Available  | `int8_t` | `1` if available, `0` if not.                         |
+
+##### `FARTTY_INTERACT_CLIP_SETDATA` ('s')
```

Comment:
also FARTTY_INTERACT_CLIP_SETDATACHUNK is forgotten, however its useful thing - in case user copies long text - it allows to send it in background, and if user copied another text while previous still sending - its possible to distrupt previous transfer and start sending new content